frontend/src/features/properties/components/forms/subforms/InformationForm.tsx

```diff
@@ -48,14 +48,17 @@ const InformationForm: FunctionComponent<InformationFormProps> = (props: Informa
       </Form.Row>
       <Form.Row>
         <Form.Label className="required">Classification</Form.Label>
-        <FastSelect
-          formikProps={formikProps}
-          disabled={props.disabled}
-          type="number"
-          placeholder="Must Select One"
-          field={withNameSpace('classificationId')}
-          options={props.classifications}
-        />
+        <Form.Group>
```

Comment:
FastSelect contains a Form.Group already, so this will result in nested form groups on a single field. Would recommend removing this unless you absolutely need it.

Reply:
working on it